x/crosschain/keeper/cctx_orchestrator.go

```diff
@@ -17,9 +18,44 @@ import (
 	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
 )
 
-// ProcessOutbound processes the finalization of an outbound transaction based on the ballot status
+func (k Keeper) ValidateOutboundZEVM(ctx sdk.Context, cctx *types.CrossChainTx) (newCCTXStatus types.CctxStatus) {
+	tmpCtx, commit := ctx.CacheContext()
+	isContractReverted, err := k.HandleEVMDeposit(tmpCtx, cctx)
```

Comment:
Idea is to have `HandleEVMDeposit` as part of the ZEVM gateway initiate outbound.
Not possible to do this?